gradio/exceptions.py

```diff
@@ -40,3 +43,44 @@ def __init__(self, message: str):
 
     def __str__(self):
         return repr(self.message)
+
+
+def log_message(message: str, level: Literal["info", "warning"] = "info"):
+    from gradio import queueing
+
+    if not hasattr(queueing.thread_data, "blocks"):  # Function called outside of Gradio
```

Comment:
If the function is called outside of Gradio (or even inside Gradio), then I think it should still raise a regular `warnings.warn()` (in the case of `gr.Warning`) and maybe log a print statement (in the case of `gr.Info`)

Reply:
We do the same for `gr.Error()` -- also raise a regular Exception in Python

Reply:
done. `raise gr.Error` always raised a regular error